mrtarget/plugins/gene/tractability.py

```diff
@@ -100,5 +101,5 @@ def build_json(self, filename=Config.TRACTABILITY_FILENAME):
                     'top_category': Category_ab
                 }
 
-                # Add data for current biomarker to self.cancerbiomarkers
+                # Add data for current gene to self.tractability
```

Comment:
Yay - updated comments!

Reply:
It is good having code review otherwise, people go Senator Palpatine's dark side

Reply:
building inline dictionaries in python is ok @afaulconbridge as the pythonic way is preferable and less verbose